client/components/List/List.jsx

```diff
@@ -20,18 +21,24 @@ function List(props) {
   return (
     <main>
       <h1>{title}</h1>
-      <div
-        className="list"
-        onClick={(e) => {
-          const target = e.nativeEvent.path.find((node) => node.tagName === 'BUTTON');
-          const args = target?.id.split('_') ?? [];
-          if (args[1] === 'ADD') {
-            onAdd(args[0]);
-          }
-        }}
-      >
-        {itemsList}
-      </div>
+      {itemsList?.length > 0 ? (
+        <div
+          className="list"
+          onClick={(e) => {
```

Comment:
@IgorChaikin Its preferred to avoid using inline function declaration inside `jsx`, try to use `useCallback` hook if handler body has no dependencies from other `jsx` variables